tests/resources/terraform/iriusrisk-tf-aws-mapping.yaml

```diff
@@ -1,12 +1,14 @@
 trustzones:
   - id:   b61d6911-338d-46a8-9f39-8dcd24abfe91
     name: Public Cloud
+    type: b61d6911-338d-46a8-9f39-8dcd24abfe91
     $default: true
 
   # SG MAPPING (AUXILIARY SG)
   # type 4
   - id:   f0ba7722-39b6-4c81-8290-a30a248bb8d9
```

Comment:
I will put on the **id** field **internet** in order not to create confusion about the same UUID on both places